### PR TITLE
Pvgenuchten add pygeoapi qs

### PR DESCRIPTION
**Pull request by cvvergara**
This PR contains #638 and #640 
Also updates the locale files
When merged the PR's will close automatically

### CODE REVIEW

File: projects_info.csv [hunk 1]
```diff
@@ -41,9 +41,10 @@ Y | zoo-project | 1.7.0 | Y | Y | Web Services | | Web Processing Service | zoo-
 Y | t-rex | 0.9.0 | Y | Y | Web Services | | Vector Tile Service | t-rex | https://t-rex.tileserver.ch/ | | t-rex | Y
 Y | actinia | 0.2.1 | Y | Y | Web Services | | Processing of Geographical Data | actinia_core | https://actinia.mundialis.de/ | | Actinia | Y
 Y | re3gistry | 2.0.0 | Y | Y | Web Services | | Management of reference codes | re3gistry | https://github.com/ec-jrc/re3gistry | | Re3gistry | Y
+Y | pygeoapi | 0.8.0 | Y | Y | Web Services | | OGC Web Services | pygeoapi | https://pygeoapi.io/ | OSGeo_community | pygeoapi | Y
```
Comment (cvvergara) [inline projects_info.csv:44]: I it on the Webservices  section

File: doc/quickstart/pygeoapi_quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+Configure new service
+================================================================================
+
+You'll now deploy a vector dataset in pygeoapi and expose its features as OGC API - Features. OGC API - Features is a standard of OGC to expose vector features as a webservice. OGC API - Features is the OpenAPI alternative to WFS.
+
+pygeoapi supports a wide range of backends to use as vector source, e.g. geopackage, PostGIS, Elasticsearch, WFS, etc. Via the GDAL OGR provider an almost unlimited set of other vector sources can be added.
+
+Interesting datasets to expose as OAPI-Features consist of real world data having multiple attributes and links to other resources.
+OSGeo Live contains a number of datasets we can use. An example is ``/var/www/html/openlayers/examples/data/geojson/countries.geojson``.
```
Comment (astroidex) [inline doc/quickstart/pygeoapi_quickstart.rst:55]: OSGeoLive not OSGeo LIve

**Review by astroidex — CHANGES_REQUESTED** [review]
Ichecked the pygeoapi documents. They look very fine. Only one OSGeoLive typo found.

**Review by Bakaniko — APPROVED** [review]

**Comment by cvvergara** [comment]
Merging as the related failed link is not part of this PR

**Pull request merged**